lib/surface/components/link.ex

```diff
@@ -2,8 +2,7 @@ defmodule Surface.Components.Link do
   @moduledoc """
   Generates a link to the given URL.
 
-  Provides similar capabilities to Phoenix's built-in `link/2`
-  function.
+  Provides similar capabilities to Phoenix's built-in `link/2` function.
```

Comment:
```suggestion
  Provides similar capabilities to Phoenix's built-in `button/2` function.
```

Reply:
Wrong file or am I not seeing something?

Reply:
Forget this suggestion. I thought I deleted it.